perun-wui-profile/src/main/java/cz/metacentrum/perun/wui/profile/pages/personal/PersonalView.ui.xml

```diff
@@ -53,6 +53,7 @@
 		<b:Modal title="Verify preferred e-mail" ui:field="verifyEmailModal" closable="true" fade="true" dataBackdrop="STATIC" dataKeyboard="true" b:id="verifyEmailModal">
 			<b:ModalBody>
 				<p:PerunLoader ui:field="verifyEmailLoader" />
+				<b:Alert dismissable="false" ui:field="mailChangeError" type="INFO" visible="false" />
```

Comment:
This should have definitely a different color so it is clear, that something has failed. The `INFO` is usually used for some neutral messages.
```suggestion
				<b:Alert dismissable="false" ui:field="mailChangeError" type="DANGER" visible="false" />
```